packages/config/config/devices/0x0330/zv2833pac.json

```diff
@@ -0,0 +1,211 @@
+				},
+				{
+					"label": "3-Way switch",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Duration of the Multilevel Start Level Change Up/Down",
+			"description": "Set the duration of the Multilevel Start Level Change Up/Down",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 8
+		},
+		{
+			"#": "4",
+			"label": "Associations in Z-Wave network Security Mode",
+			"description": "Associations in Z-Wave network Security Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15
+		},
+		{
+			"#": "5",
+			"label": "Scenes sent to the controller for button 1",
+			"description": "Scenes sent to the controller for button 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"options": [
+				{
+					"label": "button short pressed once",
+					"value": 1
+				},
+				{
+					"label": "buton short pressed twice",
+					"value": 2
+				},
+				{
+					"label": "button pressed and held",
+					"value": 4
+				},
+				{
+					"label": "button released",
+					"value": 8
+				}
+			]
+		},
+		{
+			"#": "6",
+			"label": "Scenes sent to the controller for button 2",
+			"description": "Scenes sent to the controller for button 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"options": [
+				{
+					"label": "button short pressed once",
+					"value": 1
+				},
+				{
+					"label": "button short pressed twice",
+					"value": 2
+				},
+				{
+					"label": "button pressed and held",
+					"value": 4
+				},
+				{
+					"label": "button released",
+					"value": 8
+				}
+			]
+		},
+		{
+			"#": "7",
+			"label": "Scenes sent to the controller for button 3",
+			"description": "Scenes sent to the controller for button 3",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"options": [
+				{
+					"label": "button short pressed once",
+					"value": 1
+				},
+				{
+					"label": "button short pressed twice",
+					"value": 2
+				},
+				{
+					"label": "button pressed and held",
+					"value": 4
+				},
+				{
+					"label": "button released",
+					"value": 8
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Scenes sent to the controller for button 4",
+			"description": "Scenes sent to the controller for button 4",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"options": [
+				{
+					"label": "button short pressed once",
+					"value": 1
+				},
+				{
+					"label": "button short pressed twice",
+					"value": 2
+				},
+				{
+					"label": "button pressed and held",
+					"value": 4
+				}
+			]
+		},
+		{
+			"#": "9",
+			"label": "Function of the connected switches",
+			"description": "Function of the connected switches",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "function 1",
```

Comment:
is there an explanation what functions 1-3 mean?

Reply:
Like comment above the functions mean something different based on the association groups and run several paragraphs